src/main/java/com/rog/EShop/controllers/CategoryController.java

```diff
@@ -0,0 +1,29 @@
+package com.rog.EShop.controllers;
+
+import com.rog.EShop.entity.Category;
+import com.rog.EShop.services.CategoryService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = "api/")
```

Comment:
`@RequestMapping(path = "api/")` can be simplified to `@RequestMapping("/api")`.
Same applies to `@Get/Post/Put/DeleteMapping` annotations
path `/api` is more correct than `api/`